pentaho-js/src/main/javascript/cdf-legacy/components/table.js

```diff
@@ -249,7 +252,13 @@ var TableComponent = UnmanagedComponent.extend({
         var handler = this.getSuccessHandler(success);
 
         this.queryState.setAjaxOptions({async:true});
-        this.queryState.fetchData(this.parameters == undefined ? [] : this.parameters, handler, this.failureCallback);
+        var params = this.parameters == undefined ? [] : this.parameters;
```

Comment:
Shouldn't we be also modifying `core.js`'s `getSuccessHandler` and `getErrorHandler`?